tests/libnet/cluster.go

```diff
@@ -50,7 +50,7 @@ func getPodByKubeVirtRole(virtClient kubecli.KubevirtClient, kubevirtPodRole str
 
 func SkipIpv6TestsOnSingleStackCluster(virtClient kubecli.KubevirtClient) {
 	isClusterDualStack, err := IsClusterDualStack(virtClient)
-	Expect(err).NotTo(HaveOccurred(), "should have been able to infer if the cluster is dual stack")
+	ExpectWithOffset(1, err).NotTo(HaveOccurred(), "should have been able to infer if the cluster is dual stack")
```

Comment:
i dont see a reason to have this in a separated commit,
its a code that this PR added
same for the other lines in this commit from what i understand

Reply:
done